src/App/src/Factory/IndexHandlerFactory.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Light\App\Factory;
+
+use Light\App\Handler\IndexHandler;
+use Mezzio\Template\TemplateRendererInterface;
+use Psr\Container\ContainerExceptionInterface;
+use Psr\Container\ContainerInterface;
+use Psr\Container\NotFoundExceptionInterface;
+
+use function assert;
+
+class IndexHandlerFactory
+{
+    /**
+     * @throws NotFoundExceptionInterface
+     * @throws ContainerExceptionInterface
```

Comment:
```suggestion
     * @throws ContainerExceptionInterface
     * @param class-string $requestedName
```
Maybe this fixes the error.